src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -122,7 +122,7 @@ void setup_in_use_selection_flags()
             auto loadedObj = objectMgr.GetLoadedObject(static_cast<ObjectType>(objectType), i);
             if (loadedObj != nullptr)
             {
-                Editor::SetSelectedObject(static_cast<ObjectType>(objectType), i, OBJECT_SELECTION_FLAG_2);
+                Editor::SetSelectedObject(static_cast<ObjectType>(objectType), i, OBJECT_SELECTION_FLAG_SELECTED);
```

Comment:
I'm just confused about this one `OBJECT_SELECTION_FLAG_2`. Was it denoting `SELECTED` + something else, or just `selected`? And if so, why did it have a different name? Maybe we lost some property there?